src/Lightning.ts

```diff
@@ -59,8 +63,11 @@ export const delay = (currency) => {
 
 export const LightningMixin = (superclass) =>
   class extends superclass {
+    readonly config: UserWalletConfig
+
     constructor(...args) {
       super(...args)
+      this.config = args[0].config
```

Comment:
Minor improvement. Maybe give `args[0]` a name here for better readability.

Reply:
Agreed, it looks like the eventual instantiations only ever have one object argument right now anyway ([here](https://github.com/GaloyMoney/galoy/blob/67e50d07b686f2d2a4c6a5e5ed01ab007986ce66/src/walletFactory.ts#L22), and [here](https://github.com/GaloyMoney/galoy/blob/67e50d07b686f2d2a4c6a5e5ed01ab007986ce66/src/walletFactory.ts#L36)).

Were you all expecting multiple arguments to those classes later on?

Reply:
I think going with 1 argument that is typed with a `XxxConstructorArgs` type and de-structuring via the `{fieldA, fieldB}` type syntax is a good pattern to go with. Using this `varargs` approach rarely makes sense.

Reply:
Done in #424